pkgs/top-level/all-packages.nix

```diff
@@ -9169,7 +9169,16 @@ with pkgs;
 
   ffcast = callPackage ../tools/X11/ffcast { };
 
-  fflas-ffpack = callPackage ../development/libraries/fflas-ffpack {};
+  fflas-ffpack = callPackage ../development/libraries/fflas-ffpack {
+    blas = if stdenv.isDarwin then {
```

Comment:
I think this is a bit confusing since `blas` is supposed to be a package. Adding a linkName as a passthru attribute to the packages and using `blas.linkName or "blas"` instead seems much nicer.

Reply:
Oh, a passthru attribute is exactly what we want. Didn't know about that!

Reply:
As always, please add a comment about the purpose of the `linkName` attribute. Otherwise these things tend to stay around forever even after nobody uses them anymore because nobody knows what it was good for.